### DISCUSSION TITLE
Individual Avoidances for Route Segments

### DISCUSSION THREAD

**Discussion by GuG1050** [devemux86/cruiser#425] [Q&A] [answered]
Probably it was already discussed here somewhere:
Is it planned to have  the possibility to set individual avoidances for route segments between shaping or via points (like Kurviger offers)? 
That would make planning easier as e.g. I could allow motorways for the first fast section of my trip before the curvy and individually crafted main part starts.

**Comment by devemux86** [comment, marked as answer]
@GuG1050 thanks for the suggestion.

It can be part of:
- https://github.com/devemux86/cruiser/discussions/142

---

> (like Kurviger offers)?

No, they only have different profiles per segment (straight line, fastest, curvy), not different avoid options.
Their avoid options are for the whole route, all route segments.

Cruiser already offers different profiles per route segment with BRouter (offline) for the straight line profile.
